diff --git a/manifests/backend/iscsi.pp b/manifests/backend/iscsi.pp
index cc6cb2e..86da186 100644
--- a/manifests/backend/iscsi.pp
+++ b/manifests/backend/iscsi.pp
@@ -49,6 +49,12 @@ define cinder::backend::iscsi (
     }
 
     'lioadm': {
+      service { 'target':
+        ensure  => running,
+        enable  => true,
+        require => Package['targetcli'],
+      }
+
       package { 'targetcli':
         ensure => present,
         name   => $::cinder::params::lio_package_name,
