diff --git a/manifests/compute/libvirt.pp b/manifests/compute/libvirt.pp
index 5fd9f40..fe6ea33 100644
--- a/manifests/compute/libvirt.pp
+++ b/manifests/compute/libvirt.pp
@@ -141,7 +141,7 @@ class nova::compute::libvirt (
   }
 
   nova_config {
-    'DEFAULT/compute_driver':   value => 'libvirt.LibvirtDriver';
+    'DEFAULT/compute_driver':   value => 'nova.virt.libvirt.LibvirtDriver';
     'DEFAULT/vncserver_listen': value => $vncserver_listen;
     'libvirt/virt_type':        value => $libvirt_virt_type_real;
     'libvirt/cpu_mode':         value => $libvirt_cpu_mode_real;
