diff --git a/manifests/params.pp b/manifests/params.pp
index 3e2ab4b..73ad0df 100644
--- a/manifests/params.pp
+++ b/manifests/params.pp
@@ -29,20 +29,45 @@ class mysql::params {

   case $::osfamily {
     'RedHat': {
-      if $::operatingsystem == 'Fedora' and (is_integer($::operatingsystemrelease) and $::operatingsystemrelease >= 19 or $::operatingsystemrelease == "Rawhide") {
+      case $::operatingsystem {
+        'Fedora': {
+          if is_integer($::operatingsystemrelease) and $::operatingsystemrelease >= 19 or $::operatingsystemrelease == "Rawhide" {
+            $provider = 'mariadb'
+          } else {
+            $provider = 'mysql'
+          }
+        }
+        'RedHat': {
+          if $::operatingsystemrelease >= 7 {
+            $provider = 'mariadb'
+          } else {
+            $provider = 'mysql'
+          }
+        }
+        default: {
+          $provider = 'mysql'
+        }
+      }
+
+      if $provider == 'mariadb' {
         $client_package_name = 'mariadb'
         $server_package_name = 'mariadb-server'
+        $service_name = 'mariadb'
+        $log_error           = '/var/log/mariadb/mariadb.log'
+        $config_file         = '/etc/my.cnf.d/server.cnf'
+        $pidfile             = '/var/run/mariadb/mariadb.pid'
       } else {
         $client_package_name = 'mysql'
         $server_package_name = 'mysql-server'
+        $service_name = 'mysqld'
+        $log_error           = '/var/log/mysqld.log'
+        $config_file         = '/etc/my.cnf'
+        $pidfile             = '/var/run/mysqld/mysqld.pid'
       }
+
       $basedir               = '/usr'
       $datadir               = '/var/lib/mysql'
-      $service_name          = 'mysqld'
       $socket                = '/var/lib/mysql/mysql.sock'
-      $pidfile               = '/var/run/mysqld/mysqld.pid'
-      $config_file           = '/etc/my.cnf'
-      $log_error             = '/var/log/mysqld.log'
       $ruby_package_name     = 'ruby-mysql'
       $ruby_package_provider = 'gem'
       $python_package_name   = 'MySQL-python'
