diff --git a/manifests/db/mysql.pp b/manifests/db/mysql.pp
index f883283..b042b3d 100644
--- a/manifests/db/mysql.pp
+++ b/manifests/db/mysql.pp
@@ -45,8 +45,8 @@ class nova::db::mysql(
   $dbname        = 'nova',
   $user          = 'nova',
   $host          = '127.0.0.1',
-  $charset       = 'latin1',
-  $collate       = 'latin1_swedish_ci',
+  $charset       = 'utf8',
+  $collate       = 'utf8_unicode_ci',
   $allowed_hosts = undef,
   $mysql_module  = '0.9',
   $cluster_id    = undef
