From 07a7e6a18aadd4fbd53b500595f89457f1dfa60b Mon Sep 17 00:00:00 2001
From: Martin Magr <mmagr@redhat.com>
Date: Wed, 25 Jun 2014 17:29:57 +0200
Subject: [PATCH] Don't manage RabbitMQ repos

Fixes: rhbz#1112853
---
 manifests/params.pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manifests/params.pp b/manifests/params.pp
index 9018bb7..21ddeda 100644
--- a/manifests/params.pp
+++ b/manifests/params.pp
@@ -52,7 +52,7 @@ class rabbitmq::params {
   $management_port            = '15672'
   $package_apt_pin            = ''
   $package_gpg_key            = 'http://www.rabbitmq.com/rabbitmq-signing-key-public.asc'
-  $manage_repos               = true
+  $manage_repos               = false
   $service_ensure             = 'running'
   $service_manage             = true
   #config
-- 
1.9.3

